src/test/java/com/google/firebase/auth/FirebaseTokenVerifierImplTest.java

```diff
@@ -216,6 +218,50 @@ public void testMalformedToken() throws Exception {
     tokenVerifier.verifyToken("not.a.jwt");
   }
 
+  @Test
+  public void testVerifyTokenDifferentTenantIds() throws Exception {
+    try {
+      fullyPopulatedBuilder()
+        .setTenantId("TENANT_1")
+        .build()
+        .verifyToken(createTokenWithTenantId("TENANT_2"));
+    } catch (FirebaseAuthException e) {
+      assertEquals(FirebaseUserManager.TENANT_ID_MISMATCH_ERROR, e.getErrorCode());
+      assertEquals(
+          "The tenant ID ('TENANT_2') of the token did not match the expected ('TENANT_1') value",
```

Comment:
the expected value ('TENANT_1')

here and below

Reply:
Done.